packages/sodium/src/js/modules/AmazonPrimeVideoTypeHandler.js

```diff
@@ -80,7 +80,7 @@ export default class AmazonPrimeVideoTypeHandler extends GeneralTypeHandler {
             const main = Array
                 .from(document.querySelectorAll('video'))
                 .find(e => /^blob:http\S?:\/\//.test(e.src));
-            return main.style.visibility !== "visible";
+            return main.style.visibility !== "visible" && main.style.visibility !== "";
```

Comment:
CSS visibility による可視か不可視かの判定という意図としては
```suggestion
            return getComputedStyle(main).visibility !== "visible";
```
のほうがより適切ですかねぇ

Reply:
getComputedStyle()に変更しました。
が、再度確認したところ、visibilityが空文字ではなく"visible"が入るように戻っていて、昨日のは結局なんだったのかよくわからないです。

Reply:
Amazon側の振る舞いに依存するのでなかなか厄介な問題ですけども、とりあえず問題ないということであれば良かったです
このままでも機能としては問題無さそうに見えるのでrevertするか否かはおまかせします

Reply:
master最新のextension版でも記録されることが確認できたので、一時的な現象だったのかもしれません。
android版でも問題なければマージせず閉じようと思います。

Reply:
androidだとやっぱりダメで、getComputedStyle()で取ったvisibilityでないとダメなようですね。

![image](https://user-images.githubusercontent.com/2013010/88611762-b2ca6f00-d0c4-11ea-8ffd-135ea0cfd7f9.png)

Reply:
getComputedStyle()のvisibilityで記録できるようになったのは確認できました。
cmが検知できるかどうかは未確認です。